lambda-load-old-data/install_sql.sh

```diff
@@ -0,0 +1,12 @@
+
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Nice! You found a way to incorporate this, did you test running the Dockerfile to see if it works?